src/general/NewGameSettings.tscn

```diff
@@ -1190,54 +1192,54 @@ __meta__ = {
 margin_left = 104.0
 margin_right = 104.0
 margin_bottom = 35.0
-NodeToGiveFocusTo = NodePath("../Confirm")
+NodeToGiveFocusTo = NodePath("../Start")
 
-[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+[node name="Basic" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+visible = false
 margin_left = 108.0
-margin_right = 180.0
+margin_right = 234.0
 margin_bottom = 35.0
-size_flags_horizontal = 3
-custom_styles/separator = SubResource( 1 )
+rect_min_size = Vector2( 100, 35 )
+hint_tooltip = "BASIC_VIEW_BUTTON_TOOLTIP"
+focus_neighbour_left = NodePath("../Back")
+focus_neighbour_right = NodePath("../Advanced")
+focus_next = NodePath("../Advanced")
+focus_previous = NodePath("../Back")
+text = "BASIC_VIEW"
+__meta__ = {
+"_edit_use_anchors_": false
+}
 
 [node name="Advanced" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
-margin_left = 184.0
-margin_right = 359.0
+margin_left = 108.0
+margin_right = 283.0
 margin_bottom = 35.0
 rect_min_size = Vector2( 100, 35 )
 hint_tooltip = "ADVANCED_VIEW_BUTTON_TOOLTIP"
 focus_neighbour_left = NodePath("../Basic")
-focus_neighbour_right = NodePath("../Confirm")
-focus_next = NodePath("../Confirm")
+focus_neighbour_right = NodePath("../Start")
+focus_next = NodePath("../Start")
 focus_previous = NodePath("../Basic")
 text = "ADVANCED_VIEW"
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Basic" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
-visible = false
-margin_left = 290.0
-margin_right = 416.0
+[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+margin_left = 287.0
+margin_right = 473.0
 margin_bottom = 35.0
-rect_min_size = Vector2( 100, 35 )
-hint_tooltip = "BASIC_VIEW_BUTTON_TOOLTIP"
-focus_neighbour_left = NodePath("../Back")
-focus_neighbour_right = NodePath("../Advanced")
-focus_next = NodePath("../Advanced")
-focus_previous = NodePath("../Back")
-text = "BASIC_VIEW"
-__meta__ = {
-"_edit_use_anchors_": false
-}
+size_flags_horizontal = 3
+custom_styles/separator = SubResource( 1 )
 
-[node name="Confirm" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
-margin_left = 363.0
+[node name="Start" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer"]
+margin_left = 477.0
 margin_right = 550.0
 margin_bottom = 35.0
 hint_tooltip = "CONFIRM_NEW_GAME_BUTTON_TOOLTIP"
 focus_neighbour_left = NodePath("../Advanced")
 focus_previous = NodePath("../Advanced")
-text = "CONFIRM_NORMAL"
+text = "START"
```

Comment:
I think the key here should really be "START_GAME" to be clear what it is for, even if the English text is not as clear.

Reply:
I think the context is already clear, Start Game would make it too verbose.

Reply:
The context is not clear, in Finnish translation "start" cannot be translated without context. That's why this translation key has to be "START_GAME" and for the record I'd prefer for the English text to be "Start Game" to be super clear. This is a very important button so having longer text on it, I think is actually a benefit.

Reply:
Okay, changed.